src/main/java/duke/exception/DukeException.java

```diff
@@ -16,6 +16,8 @@
     public static final String MESSAGE_COMMAND_PARAM_MISSING = "This command needs /%s to be given!";
     public static final String MESSAGE_COMMAND_PARAM_DUPLICATE = "/%s cannot be specified twice!";
     public static final String MESSAGE_COMMAND_NAME_UNKNOWN = "I don't know what %s is!";
+    public static final String MESSAGE_SORT_CRITERIA_INVALID = "%s is not a valid sort criteria!";
```

Comment:
best to name these with EXPENSE in the name as the error messages might be different for the other lists; not naming them differently would make it different to provide different messages in the future. Minor issue; probably won't be relevant in the future, but should change for consistency